train_vnext/training.py

```diff
@@ -13,10 +13,10 @@
 
 CONFIG_PATH = os.environ.get('ALCONFIG', None)
 
-def train(config, num_images):
+def train(config):
 
```

Comment:
@samiyaakhtar We can reference the legacy config somewhere. Building from the previous comment:
```
legacy_config = get_legacy_config(LEGACY_CONFIG_PATH)
...
legacy_config["tagged_output"]
legacy_config["tagging_output"]
legacy_config["image_dir"]
legacy_config["train_dir"]
...
```

We would need to import [this](https://github.com/CatalystCode/active-learning-detect/blob/master/train_vnext/validate_config.py) file

Reply:
Is the legacy config the one in the root folder called config.ini?